src/stylesheets/components/_list.scss

```diff
@@ -4,10 +4,12 @@
     @include u-line-height('body', 5);
     @include u-margin-bottom(1);
     @include u-measure($theme-text-measure-wide);
+    @include u-font('sans', 'xs');
   }
 
   svg {
     @include u-margin-right(1);
+    @include u-font('sans', 'lg');
```

Comment:
If this needs to be hard set there should be a note that icon size in lists differs from Font Awesome. Additionally, please check to make sure this approach doesn't have any unintended consequences. For example, I'm currently seeing it breaking the external link icon: 
<img width="1347" alt="Screen Shot 2020-08-24 at 10 10 12 AM" src="https://user-images.githubusercontent.com/63304927/91055218-5b9bb980-e5f2-11ea-8502-4e7a9fbbffc4.png">

Reply:
good catch! I just pushed a commit that targets the icon list icons